### ISSUE TITLE
[4.0] [a11y] Setting icon is not accessible

### ISSUE THREAD

**Issue by Harshil99** [joomla/joomla-cms#24325]
### Steps to reproduce the issue
Go to Control Panel.
Tab through whole page using keyboard.  

### Expected result
Setting Icon of all Administrator Modules (Sample Data, Logged-in Users etc.) can be selected for keyboard only users.

### Actual result
Not able to select Setting icon through keyboard.

### System information (as much as possible)
Joomla! 4.0.0-alpha8-dev
Google Chrome Version 72.0.3626.121 (Official Build) (64-bit)
Ubuntu 18.04.2 LTS

**Comment by Puneet-Jain-18** [comment]
works well for Joomla! 4.0.0-alpha7-dev
Problem is in alpha 8

**Comment by Quy** [comment]
Please test PR #24340<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24325">issues.joomla.org/tracker/joomla-cms/24325</a>.</sub>
